property-service/src/main/java/org/egov/ps/model/PurchaseDetails.java

```diff
@@ -69,5 +72,21 @@
 
 	@JsonProperty("auditDetails")
 	private AuditDetails auditDetails;
+	
+	@JsonProperty("purchaserDocuments")
```

Comment:
purchaser documents should just be like ownerDocuments. They are just a logical separation for the UI.